deployments/scripts/run-local-devnet.sh

```diff
@@ -36,8 +36,11 @@ else
     use_tui="false"
 fi
 
+# Set unique API port for controlling running services.
+export PC_PORT_NUM="8888"
+
```

Comment:
TODO: we can and should use unix sockets where possible, but I'm punting on that for now, and just using 8888/TCP for the compose API port.